hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SubProcessHapiTestNode.java

```diff
@@ -92,6 +92,8 @@ public void start() {
             builder.command(
                             javaCmd,
                             "-Dfile.encoding=UTF-8",
+                            "-agentlib:jdwp=transport=dt_socket,server=y,suspend=" + (nodeId == 0 ? "n" : "n")
```

Comment:
Worth noting that, similar to the three test profiles now prompting any test run in IDE, we could modify the debug profile in the build to set a property to control the suspend parameter here.
Future work, but might make debug for out-of-process a little smoother.